### PR TITLE
HomePage navigation tab

### PR DESCRIPTION
**Pull request by rickwolox**
### Summary
HomePage is now available with navbar on the top, you can switch between them.

### Trello Card
https://trello.com/c/2OwtIGFq/15-u4-home-page-tabs-de-navegaci%C3%B3n

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -32,6 +32,10 @@
             </intent-filter>
         </activity>
 
+        <meta-data
```
Comment (lmiotti) [inline app/src/main/AndroidManifest.xml:35]: What's this? It's really necessary?

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomePageFragment.kt [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import ar.com.wolox.android.R
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import com.google.android.material.tabs.TabLayout
+import kotlinx.android.synthetic.main.fragment_homepage.*
+
+class HomePageFragment : WolmoFragment<HomePagePresenter>(), IHomePageView {
+
+    override fun init() {
+        vTabSelection.getTabAt(0)?.let {
+            it.icon = ContextCompat.getDrawable(activity!!, getSelectIcon(0))
```
Comment (lmiotti) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/HomePageFragment.kt]: Replace `activity!!` with `requireActivity()`

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (rickwolox) [reply app/src/main/AndroidManifest.xml:35]: It was added when I import roboto font on Android Studio

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomePageFragment.kt [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        }
+    }
+
+    override fun layout(): Int {
```
Comment (lmiotti) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/HomePageFragment.kt]: Here you can replace braces with `Int = R.layout.fragment_homepage`

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomePageFragment.kt [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    }
+
+    override fun setListeners() {
+        vTabSelection.addOnTabSelectedListener(object : TabLayout.OnTabSelectedListener {
+            override fun onTabReselected(p0: TabLayout.Tab?) {
+            }
+
+            override fun onTabUnselected(p0: TabLayout.Tab?) {
+                p0?.position.let {
+                    p0?.icon = ContextCompat.getDrawable(activity!!, getUnSelectIcon(it))
```
Comment (lmiotti) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/HomePageFragment.kt]: Same as above with `activity!!`

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomePageFragment.kt [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+                }
+            }
+
+            override fun onTabSelected(p0: TabLayout.Tab?) {
+                p0?.position.let {
+                    p0?.icon = ContextCompat.getDrawable(activity!!, getSelectIcon(it))
```
Comment (lmiotti) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/HomePageFragment.kt]: Same as above

File: app/src/main/res/layout/fragment_homepage.xml [hunk 6]
```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (lmiotti) [inline app/src/main/res/layout/fragment_homepage.xml]: Move xmlns:android to new line

File: app/src/main/res/layout/fragment_homepage.xml [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <ImageView
```
Comment (lmiotti) [inline app/src/main/res/layout/fragment_homepage.xml:9]: Move xmlns:app below xmlns:android and tools below it

File: app/src/main/res/layout/fragment_homepage.xml [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+        android:fontFamily="@font/roboto_regular"
+        android:gravity="center_vertical"
+        android:text="@string/home_title_wolox"
+        android:textColor="@color/colorTopTitle"
+        android:textSize="@dimen/font_big"
+        app:layout_constraintStart_toEndOf="@+id/vLogoWolox"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+
```
Comment (lmiotti) [inline app/src/main/res/layout/fragment_homepage.xml]: Remove this empty line

**Review by lmiotti — CHANGES_REQUESTED** [review]
The PR is almost good. Some things to be done. Keep on like this!

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomePageFragment.kt [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+import androidx.core.content.ContextCompat
+import ar.com.wolox.android.R
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import com.google.android.material.tabs.TabLayout
+import kotlinx.android.synthetic.main.fragment_homepage.*
+
+class HomePageFragment : WolmoFragment<HomePagePresenter>(), IHomePageView {
+
+    override fun init() {
+        vTabSelection.getTabAt(0)?.let {
```
Comment (lmiotti) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/HomePageFragment.kt]: Solve this harcoded integer. Move it to a companion object, such as DEFAULT_TAB

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomePageFragment.kt [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+                p0?.position.let {
+                    p0?.icon = ContextCompat.getDrawable(requireActivity(), getSelectIcon(it))
+                }
+            }
+        })
+    }
+
+    private fun getSelectIcon(position: Int?): Int {
+        return when (position) {
+            0 -> R.drawable.ic_news_list_on
```
Comment (lmiotti) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/HomePageFragment.kt]: Same as above, solve this. NEWS_TAB and PROFILE_TAB consts can be created.

**Review by lmiotti — CHANGES_REQUESTED** [review]
One new correction I've missed :smile:

**Review by lmiotti — APPROVED** [review]
superheroes

**Pull request merged**